src/app/settings/containers/roles/roles.component.html

```diff
@@ -1,5 +1,6 @@
 <app-data-table
   [data$]="users$"
+  [isReadOnly]="true"
```

Comment:
What if we default data tables to readOnly and accept an optional input `isEditable`? What do you think? Your call here.

Reply:
Added to #290